nix/packages/yasunori-cli/default.nix

```
@@ -0,0 +1,6 @@
+{
+  writers,
+}:
+writers.writeNu "/bin/yasunori-cli" ''
```

```suggestion
writers.writeNuBin "yasunori-cli" ''
```
こういうときは `writeNuBin` という派生を使うと良いです